pom.xml

```diff
@@ -106,12 +107,17 @@
       <dependency>
         <groupId>com.azure.spring</groupId>
         <artifactId>azure-spring-cloud-feature-management</artifactId>
+        <version>${version.azure-spring-cloud-feature-management}</version>
+      </dependency>
+      <dependency>
+        <groupId>com.azure.spring</groupId>
+        <artifactId>spring-cloud-azure-feature-management-web</artifactId>
         <version>${version.spring-cloud-feature-management}</version>
       </dependency>
       <dependency>
         <groupId>com.azure.spring</groupId>
         <artifactId>azure-spring-cloud-feature-management-web</artifactId>
-        <version>${version.spring-cloud-feature-management}</version>
+        <version>${version.azure-spring-cloud-feature-management}</version>
```

Comment:
Is it necessary to keep both `spring-cloud-azure-feature-management-web` and `azure-spring-cloud-feature-management-web`?